src/Components/Logo/Provider.php

```diff
@@ -42,7 +42,7 @@ public function register(): void
             Artisan::starting(
                 function ($artisan) use ($config) {
                     $artisan->setName(
-                        (string) new FigletString($config->get('app.name'), $config->get('logo', []))
+                        (string) new FigletString($config->get('logo.name') ?? $config->get('app.name'), $config->get('logo', []))
```

Comment:
Ah nice. 👍🏻 That was going to be one of my comments, about falling back to `app.name` still.